rfcs/detecting_sensitive_textual_content/20230309-project_proposal_detecting_sensitive_textual_content.md

```diff
@@ -0,0 +1,379 @@
+# Project Proposal: Detecting sensitive textual content
+
+## Reviewers
+
+- [ ] @aetherunbound
+- [ ] @krysal
+
+## Project summary
+
+Detect results with sensitive textual content by checking their textual content
+against a list of sensitive terms. Once results with sensitive terms in their
+textual content are determined, blur those results on the frontend.
```

Comment:
```suggestion
Detect results with sensitive textual content by checking their textual content
against a list of sensitive terms. Once results with sensitive terms in their
textual content are determined, blur those results on the frontend.
```

I think it might also be important to note that we _are_ making an inference about image contents based on the textual content matching.

Reply:
I'll add a note to this respect.